drive/src/drive/initialization.rs

```diff
@@ -52,6 +52,10 @@ impl Drive {
 
         batch.add_insert_empty_tree(vec![], vec![RootTree::Pools as u8]);
 
+        batch.add_insert_empty_tree(vec![], vec![RootTree::WithdrawalTransactions as u8]);
+
+        batch.add_insert_empty_tree(vec![], vec![RootTree::WithdrawalTransactionsCounter as u8]);
```

Comment:
I wouldn't create an additional tree on root level just for counter. Let's have one for withdrawals which have two leaves: queue and counter.